packages/curve-ui-kit/src/shared/ui/DataTable/ResetFiltersButton.tsx

```diff
@@ -0,0 +1,27 @@
+import Button from '@mui/material/Button'
+import { t } from '@ui-kit/lib/i18n'
+import { TransitionFunction } from '@ui-kit/themes/design/0_primitives'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+
+const { Spacing } = SizesAndSpaces
+
+export const ResetFiltersButton = ({ onClick, hidden }: { onClick: () => void; hidden: boolean }) => (
+  <Button
+    color="ghost"
+    size="small"
+    onClick={onClick}
+    data-testid="reset-filter"
+    sx={{
+      minWidth: 0,
+      marginInlineStart: Spacing.xs,
+      transition: `font-size ${TransitionFunction}, padding ${TransitionFunction}, margin ${TransitionFunction}`,
+      ...(hidden && {
+        fontSize: '0 !important', // unfortunately, MUI overrides the font-size very specifically
```

Comment:
mayube three &'s do the trick here as well?

Reply:
yes